### PR TITLE
added additional links to ODE stability

### PR DESCRIPTION
**Pull request by PhiSpel**
added three links from MATHWORKS

### CODE REVIEW

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb [hunk 1]
```diff
@@ -225,6 +225,19 @@
    "source": [
     "- Variieren Sie nun `g`. Können Sie $g(t) = 4 t^2$ implementieren und die DGL lösen?"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Zusatzinformationen\n",
```
Comment (joergbrech) [inline content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb:233]: Could you make this an admonition? This fits better with the style of the book: https://jupyterbook.org/content/content-blocks.html#notes-warnings-and-other-admonitions

**Review by joergbrech — CHANGES_REQUESTED** [review]

**Comment by PhiSpel** [comment]
I did this also for Raketenstart, which I used as a template for the Zusatzinformationen

Strangely, it's formatted yellow and I do not know how to globally change the format for `:class: tip`

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb [hunk 2]
```diff
@@ -225,6 +225,20 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```{admonition} Zusatzinformationen\n",
+    ":class: tip\n",
+    "Die MATLAB Dokumentation und zugehörige Blogs liefern weitere anschauliche Beispiele:\n",
+    "\n",
+    "- https://de.mathworks.com/help/matlab/math/choose-an-ode-solver.html\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb:237]: We should stick to using hyperlinks like in the rocket launch information box

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]
Looks good to me, we can merge it after implementing the hyperlinks

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb (hunk 2 above)
Comment (PhiSpel) [reply content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb]: Oh yes, good point

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb (hunk 1 above)
Comment (PhiSpel) [reply content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb]: Done in commit ec16ded

**Review by sandro-elsweijer — APPROVED** [review]

**Pull request merged**